### PR TITLE
Sync and validate master branch on Arm64 Windows

### PR DESCRIPTION
**Pull request by eukarpov**

### CODE REVIEW

**Review by cristianPanaite — APPROVED** [review]

File: .github/workflows/sync-and-validate-master.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  schedule:
+    - cron: "0 3 * * *"
+
+jobs:
+  sync-fork:
+    name: Sync master
+    runs-on: ubuntu-latest
+
+    steps:
+    - env:
```
Comment (Blackhex) [inline .github/workflows/sync-and-validate-master.yml]: I am not sure if it's mix of spaces and tabs but the indentation seems to be weird here.

File: .github/workflows/ci-arm64.yml [hunk 2]
```diff
@@ -39,12 +51,14 @@ jobs:
         type benchmark.txt
 
   build-openssl-windows-arm64-clangcl:
-    runs-on: [self-hosted, Windows, ARM64, GCC]
+    runs-on: [self-hosted, Windows, ARM64, GCC, D2ps_v5]
```
Comment (Blackhex) [inline .github/workflows/ci-arm64.yml:54]: Can you please explain a little bit more why this job is being restricted to that particullar runner size?

**Review by Blackhex — APPROVED** [review]
Just one question, otherwise looks OK.

File: .github/workflows/sync-and-validate-master.yml (hunk 1 above)
Comment (eukarpov) [reply .github/workflows/sync-and-validate-master.yml]: a good catch. Fixed

File: .github/workflows/ci-arm64.yml (hunk 2 above)
Comment (eukarpov) [reply .github/workflows/ci-arm64.yml:54]: It is needed to be sure that benchmark is executed on the same hardware scale

Comment (Blackhex) [reply .github/workflows/ci-arm64.yml:54]: Then I'd suggest replacing this label with some that wouldn't need to be changed when we decide to run it on different machine, e.g. `OpenSSL`.

Comment (eukarpov) [reply .github/workflows/ci-arm64.yml:54]: Yes, it is also option, however less flexible for different hardware tests.

**Pull request merged**